client/post-editor/editor-ground-control/history-button.jsx

```diff
@@ -12,23 +12,37 @@ import { flow } from 'lodash';
  * Internal dependencies
  */
 import { openPostRevisionsDialog } from 'state/posts/revisions/actions';
-
+import { recordTracksEvent } from 'state/analytics/actions';
 import EditorRevisionsDialog from 'post-editor/editor-revisions/dialog';
 
-const HistoryButton = ( { loadRevision, postId, siteId, openDialog, translate } ) => (
-	<div className="editor-ground-control__history">
-		<button className="editor-ground-control__history-button button is-link" onClick={ openDialog }>
-			{ translate( 'History' ) }
-		</button>
-		<EditorRevisionsDialog loadRevision={ loadRevision } postId={ postId } siteId={ siteId } />
-	</div>
-);
+class HistoryButton extends PureComponent {
```

Comment:
Fun fact! `connect()`ed components are already pure by default unless we explicitly set them not to be, so we can avoid `PureComponent` and just stick with `Component` and save some overhead

Reply:
Sounds good! Changed to Component in https://github.com/Automattic/wp-calypso/pull/20797/commits/14bed9d9cfe97cdad3b9a79bf1a74299523ff24a